### DISCUSSION TITLE
梅林 arm380 除了第一次运行有日志外，后面都没有运行日志了，不管怎么动它

### DISCUSSION THREAD

**Discussion by upsetterlesson** [messense/aliyundrive-webdav#187] [Q&A]
[![](https://cdn.jsdelivr.net/gh/upsetterlesson/picqiu/images/Snipaste_2021-12-07_07-33-16.png)](url)

**Comment by upsetterlesson** [comment]
[![](https://cdn.jsdelivr.net/gh/upsetterlesson/picqiu/images/20211207084306.png)](url)

**Comment by messense** [comment]
手头没有 arm380 的梅林设备了，不好 debug.

**Comment by upsetterlesson** [comment]
远程设备可以debug吗
